src/components/RegionSelect/RegionSelect.module.scss

```
@@ -124,9 +124,12 @@
   margin: theme.$spacingXSmall;
   left: 0 !important;
   max-width: calc(100% - theme.$spacingXSmall * 2);
-  @include theme.mobileUpBreakPoint {
+  // TEMP (C235): anchor dropdown to the repositioned breadcrumb (left: 56px).
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)